metecho/api/jobs.py

```diff
@@ -741,3 +752,18 @@ def create_gh_branch_for_new_project(project, *, user):
 
 
 create_gh_branch_for_new_project_job = job(create_gh_branch_for_new_project)
+
+
+def available_task_org_config_names(project, *, user):
+    repo_id = project.get_repo_id(user)
+    with local_github_checkout(user, repo_id) as repo_root:
+        config = get_org_config(user=user, repo_id=repo_id, project_path=repo_root)
+        project.available_task_org_config_names = [
+            {"key": key, **value} for key, value in config.orgs__scratch.items()
+        ]
```

Comment:
I think this will work -- though it will potentially pull in lots of info from cumulusci.yml that we don't actually need.

Reply:
I can constrain it more, but I figured this would be useful.